docs/apache-airflow/installation/upgrading.rst

```diff
@@ -57,6 +57,10 @@ when you choose to upgrade airflow via their UI.
 How to upgrade
 ==============
 
+Reinstall Apache Airflow™, specifying the desired new version. For example, in the case of a bootstrapped
+local instance, this can be accomplished easily with the ``AIRFLOW_VERSION`` environment variable. For more
```

Comment:
I am not sure if I get it - the ``AIRFLOW_VERSION`` needs to be used somehow - just setting it will not upgrade / reinstall airflow automatically. Could you elaborate a bit more ?

Maybte you are referring to some specific tooling that is doing such upgrade automatically?)

Reply:
Thanks @potiuk , added more details here.